test/libsolidity/ABIEncoderTests.cpp

```diff
@@ -779,7 +779,7 @@ BOOST_AUTO_TEST_CASE(struct_in_constructor_data_short)
 
 	NEW_ENCODER(
 		BOOST_CHECK(
-			compileAndRunWithoutCheck({{"", sourceCode}}, 0, "C", encodeArgs(0x20, 0x60, 0x03, 0x80, 0x00)).empty()
+			compileAndRunWithoutCheck({{"", sourceCode}}, nullopt, 0, "C", encodeArgs(0x20, 0x60, 0x03, 0x80, 0x00)).empty()
```

Comment:
I think I will move that "file" parameter somewhere to the end and set a default value, with this these tests don't need to be changed.